modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -231,10 +237,24 @@ export default class Tile3DLayer extends CompositeLayer {
     } = tileHeader.content;
     const {positions, normals, colors} = attributes;
 
-    return (
-      positions &&
-      new PointCloudLayer({
-        id: `3d-point-cloud-tile-layer-${tileHeader.contentUri}`,
+    if (!positions) {
+      return null;
+    }
+
+    const {getColor} = this.props;
```

Comment:
`getColor` is not defined in `defaultProps`. What input is it supposed to receive?

Reply:
Oops. Added it to defaultProps